api/v2/api.go

```diff
@@ -146,19 +144,27 @@ func (api *API) getStatusHandler(params general_ops.GetStatusParams) middleware.
 		Config: &open_api_models.AlertmanagerConfig{
 			Original: &original,
 		},
-		Cluster: &open_api_models.ClusterStatus{
-			Name:   &name,
-			Status: &status,
-			Peers:  []*open_api_models.PeerStatus{},
-		},
 	}
 
-	for _, n := range api.peer.Peers() {
-		address := n.Address()
-		resp.Cluster.Peers = append(resp.Cluster.Peers, &open_api_models.PeerStatus{
-			Name:    &n.Name,
-			Address: &address,
-		})
+	// If api.peers == nil, Alertmanager cluster feature is disabled.
```

Comment:
I would reverse this comment. "If alertmanager cluster feature is disabled, then api.peers == nil".